scripts/bulk-parse.js

```diff
@@ -0,0 +1,62 @@
+const parser = require('./parse-json.js');
```

Comment:
Not a required change, but you don't need the `.js`, just `require('./parse-json')` will work the same.